### PR TITLE
8 commits were combined into 1 commit.

### PR DESCRIPTION
**Pull request by ViaSocket-Git**

### CODE REVIEW

File: src/components/main/poweredByTechdoc/poweredByTechdoc.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        .fetti {
+            position: absolute;   
+     
+            width: 10px;
+            height: 10px;
+            border-radius: 50%;
+        }
+
+        > :first-child {
+            & :first-child {
```
Comment (Saurabh186) [inline src/components/main/poweredByTechdoc/poweredByTechdoc.scss:16]: ....

**Review by Saurabh186 — CHANGES_REQUESTED** [review]

File: src/components/apiDocReview/apiDocReview.scss [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
-  color:#adadad;
-  gap: 20px;
+.feedback-saved {
+  text-align: center;
+  color: green;
+  font-weight: light;
 }
+
+.enpoint-title {
+  & :first-child {
```
[the remark this answers is not in the document]
Comment (ViaSocket-Git) [reply src/components/apiDocReview/apiDocReview.scss:22]: Resolved

File: src/components/apiDocReview/apiDocReview.scss [hunk 3]
```diff
@@ -1,23 +1,30 @@
...
-  .feedback-saved {
-    text-align: center; 
-    color: green;       
-    font-weight: light; 
-  }
-  .like-unline {
-  color:#adadad;
-  gap: 20px;
+.feedback-saved {
+  text-align: center;
```
[the remark this answers is not in the document]
Comment (ViaSocket-Git) [reply src/components/apiDocReview/apiDocReview.scss:16]: Resolved

File: src/components/main/poweredByTechdoc/poweredByTechdoc.scss (hunk 1 above)
Comment (ViaSocket-Git) [reply src/components/main/poweredByTechdoc/poweredByTechdoc.scss:16]: Resolved

**Review by Saurabh186 — APPROVED** [review]

**Pull request closed without merging**